modules/tracking/test/test_trackers.cpp

```diff
@@ -567,6 +567,24 @@ TEST_P(DistanceAndOverlap, Scaled_Data_CSRT)
   test.run();
 }
 
+TEST(GOTURN, memory_usage)
+{
+  cv::Rect2d roi(145, 70, 85, 85);
+  cv::Mat frame;
+
+  cv::Ptr<cv::Tracker> tracker = cv::TrackerGOTURN::create(TrackerGOTURN::Params{cvtest::findDataFile("../dnn/gsoc2016-goturn/goturn.prototxt"), cvtest::findDataFile("../dnn/gsoc2016-goturn/goturn.caffemodel")});
```

Comment:
Better use
```cpp
  cv::Ptr<cv::Tracker> tracker = cv::TrackerGOTURN::create();
tracker->read(...)
```
https://github.com/opencv/opencv_contrib/blob/229653a4819397635f50073ac7a627a885028392/modules/tracking/src/gtrTracker.cpp#L101-L104

model paths should be the fields of FileNode

Reply:
But previously, before this patch, they were just ```String``` type in GOTURN tracker. Can you, please, explain the difference here?

Reply:
Instead of `Params` constructor change, you need to modify `Params::read` so it will parse `FileNode` and set `modelTxt_` and `modelBin_`